sdk/e2e/pom.xml

```diff
@@ -55,6 +55,11 @@
       <artifactId>slf4j-api</artifactId>
       <version>1.7.28</version> <!-- {x-version-update;org.slf4j:slf4j-api;external_dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-mgmt-graph-rbac</artifactId>
```

Comment:
You need to add a version tag here.

Reply:
Thanks - is there a reason why no pipeline job is running on this PR?

Reply:
This particular path doesn't have a pipeline setup for it. I suggest either adding one or adding it to run the core pipeline.